platformlibs/data_handler.py

```diff
@@ -28,25 +28,30 @@ class DataHandler(object):
     def __init__(self,
                  spark_context,
                  datasource,
-                 path):
+                 path,
+                 isTopic=False):
         """ Constructor """
         self.spark_context = spark_context
         self.datasource = datasource
+        self.is_topic = isTopic
         self.path = path
         self._rdd = None
         self._hdfs_root_uri = None
-        self.schema = self._load_schema()
+        self.schema = '''
```

Comment:
It would be more flexible to have the avro schema being read from the avro file metadata header rather than hardcoding it in the code.